client/src/pages/Login.tsx

```diff
@@ -16,24 +15,26 @@ const LoginImage = styled.div`
     }
 `;
 const LoginText = styled.div`
-    color: ${colors.green};
+    color: var(--green);
     text-align: center;
     font-size: 36px;
     font-weight: 700;
     margin-top: 30px;
     width: 100%;
 `;
-const LoginBtn = styled.button`
+const LoginBtn = styled(Link)`
     border: none;
     color: #fff;
     font-size: 18px;
     font-weight: 600;
```

Comment:
Same here.

Reply:
Here it won't work as `font` shorthand property requires at least the font-size and font-family values to be specified. here we dont have `font-family`.